CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ## 2.6.3 (released 02.10.2024)
 * Made caching optional (selectable by adding a package to your project)
-* Improve `play.PropertiesConfLoader` to ease `conf/` directory usage @xabolcs 
+* Improve `play.conf.PropertiesConfLoader` to ease `conf/` directory usage @xabolcs
```

Comment:
Unsure where this change should go (here or the cleanup commit), or just to drop. :thinking:

Reply:
PropertiesConfLoader is still in `play` package. This should be reverted!